jobsub/examples/gbl_local/config.cfg

```diff
@@ -51,10 +51,10 @@ HotpixelRunNumber	= @RunNumber@
 SkipNEvents		= 0
 
 # Output subfolder structure
-DatabasePath		= ./output/database
-HistogramPath		= ./output/histograms
-LcioPath            = ./output/lcio
-LogPath				= ./output/logs
+DatabasePath		= %(BasePath)s/output/database
```

Comment:
Do we really want this? This won't allow users who have a generic installation to run teh example in a custom directory!